genienlp/data_utils/numericalizer.py

```diff
@@ -44,23 +44,23 @@
 ALLOWED_FAST_TOKENIZERS = {
     'facebook/bart-base',
     'facebook/bart-large',
-    'google/mt5-small',
-    'google/mt5-base',
-    'google/mt5-large',
-    'google/mt5-xl',
-    'google/mt5-xxl',
     'sshleifer/bart-tiny-random'
 }
 # known NOT to work:
 # - all the BERT models
 # - all the XLM-R models
 #
-# mBART models work when preprocessing, because they're SPM/RoBERTa-based so they respect
+# mBART, t5, and mt5 models work when preprocessing, because they're SPM/RoBERTa-based so they respect
```

Comment:
t5 doesn't work at all if you don't preprocess, it will die if you try to add new tokens (at least in transformers 4.0)